### PR TITLE
Update Jetsons

### PR DESCRIPTION
**Pull request by mikepizzo**
* Update to latest RESTier V1
* Support unqualified actions/functions
* Add instructions for building/running local

### CODE REVIEW

**Comment by mikepizzo** [comment]
Updates to 1.0 GA version of Restier, and enables unqualified actions/functions.

File: samples/Jetsons/Jetsons/Jetsons.csproj [hunk 1]
```diff
@@ -35,15 +35,4 @@
 		<ProjectReference Include="..\Microsoft.Restier.Providers.InMemory\Microsoft.Restier.Providers.InMemory.csproj" />
 	</ItemGroup>
 
-	<ItemGroup>
-		<Reference Include="Microsoft.Restier.AspNetCore">
-			<HintPath>..\RESTierAssemblies\Microsoft.Restier.AspNetCore.dll</HintPath>
-			<CopyLocal>True</CopyLocal>
-		</Reference>
-		<Reference Include="Microsoft.Restier.Core">
-			<HintPath>..\RESTierAssemblies\Microsoft.Restier.Core.dll</HintPath>
```
Comment (ralfhandl) [inline samples/Jetsons/Jetsons/Jetsons.csproj:44]: Does this mean that the folder `RESTierAssemblies` and the two DLLs can be deleted?

Comment (mikepizzo) [reply samples/Jetsons/Jetsons/Jetsons.csproj:44]: Yes -- good catch; thanks!

File: samples/Jetsons/ReadMe.md [hunk 2]
```diff
@@ -5,9 +5,11 @@ The code in this directory implements a simple sample RAPID service based on the
...
 There are three project directories:
-1) Microsoft.Restier.Providers.InMemory -- Provides common utilities for registering per-session or global services
-2) Jetsons -- this is the main read-only service hosted on Azure
-3) JetsonsRW -- this is a read-write version of the Jetsons service that creates a sandbox in which the user can experiment with making updates to the data through POST/PATCH/DELETE operations.
+
+1. Microsoft.Restier.Providers.InMemory -- Provides common utilities for registering per-session or global services
+2. Jetsons -- this is the main read-only service hosted on Azure
+3. JetsonsRW -- this is a read-write version of the Jetsons service that creates a sandbox in which the user can experiment with making updates to the data through POST/PATCH/DELETE operations.
 
 ## Publish Sample Service To RAPID.ROCKS
```
[the remark this answers is not in the document]
Comment (mikepizzo) [reply samples/Jetsons/ReadMe.md:13]: Notes to self:

1. can build by installing the latest [.NET 7.x SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) from command line using:

 dotnet build C:\Repos\Rapid\odata-rapid-jetsons\samples\Jetsons/JetsonsReadWrite/JetsonsReadWrite.csproj.

2. I don't know if there is a way to run locally from the command line.

3. You can build/run locally from visual studio code (don't know what extensions need to be installed -- probably at least C# for Visual Studio Code).

[the remark this answers is not in the document]
Comment (mikepizzo) [reply samples/Jetsons/ReadMe.md:44]: Add sections on building, running local, and deploying in VS Code.

@ralfhandl -- Please verify that instructions for building/running local work (you won't be able to deploy without the azure account information).

[the remark this answers is not in the document]
Comment (ralfhandl) [reply samples/Jetsons/ReadMe.md:44]: Works!

**Review by ralfhandl — APPROVED** [review]

**Pull request merged**